### ISSUE TITLE
Distinguish new notifcations for accessibility

### ISSUE THREAD

**Issue by RickDBCN** [filamentphp/filament#4158]
### Package

filament/notifications

### Package Version

v2.2.16.11

### Laravel Version

9.x

### PHP Version

8.1.x

### Problem description

Several screens can't see the difference between unread notifications and read notifications. See: https://prnt.sc/G6sXbDlmmwRn

### Expected behavior

```bg-gray-50``` should be ```bg-primary-50``` for increased accessibility. See change: https://prnt.sc/J-bu21Zn0xJe

### Steps to reproduce

//

### Reproduction repository

//
